spec/models/question_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe Question, type: :model do
+  let(:quiz) { Quiz.create!(title: "Sample Quiz", description: "This is a sample quiz") }
+
+  describe "#correct_answer" do
+    context "when correct_option is valid" do
+      it "returns the text for option_a" do
+        question = quiz.questions.create!(
+          content: "Test Question",
+          option_a: "Correct A",
+          option_b: "Wrong",
+          option_c: "Wrong",
+          option_d: "Wrong",
+          correct_option: "option_a",
+        )
```

Comment:
```suggestion
        question = described_class.new(
          content: "Test Question",
          option_a: "Correct A",
          option_b: "Wrong",
          option_c: "Wrong",
          option_d: "Wrong",
          correct_option: "option_a",
        )
```

We can avoid the writing to the database by just interfacing with this class directly to create an object rather than writing data to the db.